transformer_engine/paddle/csrc/custom_ops.cu

```diff
@@ -1260,6 +1305,28 @@ void te_scaled_upper_triang_masked_softmax_backward(paddle::Tensor &output_grads
       softmax_results.stream());
 }
 
+__global__ void UpdateFP8MetaKernel(
```

Comment:
Regarding this issue, what I really need is a custom kernel that can utilize the identifier/parameterSetter mechanism since we need the first parameter of the kernel to be the identifier. Therefore, I reverted to the legacy implementation (see [this PR](https://github.com/NVIDIA/TransformerEngine/pull/593/files#diff-0d7172c528739ab6b144f09788506478439b3748d259466544dbea1a7f0ee18bL1192)). The identifier/parameterSetter mechanism cannot be used if we employ NVTE kernels, as I prefer to keep the complex identifier/parameterSetter mechanism confined to this CU file. Do you have any suggestions on how to achieve this better?